doc/developers_guide.txt

```diff
@@ -107,7 +107,32 @@ Coding standards and style
 --------------------------
 
 All code should conform as much as possible to `PEP 8`_, and should run with
-Python 2.6 and 2.7.
+Python 2.6 and 2.7. Lines should be no longer than 99 characters.
+
+
+Reviewing pull requests
+-----------------------
+
+All contributors are encouraged to review pull requests, and all pull requests must have at least one review before
+merging.
+
+Things to check for:
+
+  * Does the pull request implement a single, well-defined piece of functionality?
+    (pull requests which perform system-wide refactoring are sometimes necessary, but need much more careful scrutiny)
+  * Does the code work? Is the logic correct? Does is anticipate possible failure conditions (e.g. lack of internet connection)?
+  * Is the code easily understood?
+  * Does the code conform to the coding standards (see above)?
+  * Does the code implement a general solution, or is the code too specific to a particular (language|version control system|storage backend)?
+  * Do all public functions/classes have docstrings?
+  * Are there tests for all new/changed functionality?
+  * Has the documentation been updated?
+  * Has the Travis CI build passed?
+  * Is the syntax compatible with both Python 2 and 3? (even if we don't yet support Python 3, any new code should try to do so)
```

Comment:
we could look to config linters as part of tests to enforce this, see e.g., http://stackoverflow.com/questions/12450348/how-to-integrate-any-python-lint-with-github-commit-status-api